test/manipulation.test.js

```diff
@@ -1108,21 +1130,34 @@ describe('manipulation', function() {
   }
 
   var hasReplaceById = !!getSchema().connector.replaceById;
+
   bdd.describeIf(hasReplaceById, 'replaceOrCreate when forceId is true', function() {
     var Post;
+    var ds = getSchema();
     before(function(done) {
-      var ds = getSchema();
       Post = ds.define('Post', {
         title: {type: String, length: 255},
         content: {type: String},
       }, {forceId: true});
       ds.automigrate('Post', done);
     });
 
+    var id = db.generateId() || 123;
+
     it('fails when id does not exist in db', function(done) {
-      var post = {id: 123, title: 'a', content: 'AAA'};
+      // Why should replaceOrCreate fail when id does not exists in db?
+      // With Cassandra, it will pass.
```

Comment:
put the entire `forceId: true` test cases in bdd.itIf(connectorCapabilities.supportForceId